### ISSUE TITLE
Bad github URL using backslashes when running scan on Windows

### ISSUE THREAD

**Issue by TeamDman** [kubescape/kubescape#1003]
# Describe the bug

Issue running scan on Windows 10

```
{"level":"info","ts":"2023-01-03T15:09:33-05:00","msg":"Kubescape scanner starting"}
W0103 15:09:34.333850   13548 azure.go:92] WARNING: the azure auth plugin is deprecated in v1.22+, unavailable in v1.26+; use https://github.com/Azure/kubelogin instead.
To learn more, consult https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins
E0103 15:09:35.128272   13548 azure.go:162] Failed to acquire a token: storing the refreshed token in configuration: persisting the configuration: open C:\Users\Teamy\.kube\config.lock: The file exists.
E0103 15:09:36.453231   13548 azure.go:162] Failed to acquire a token: storing the refreshed token in configuration: persisting the configuration: open C:\Users\Teamy\.kube\config.lock: The file exists.
{"level":"warn","ts":"2023-01-03T15:09:36-05:00","msg":"failed to create host scanner: in NewHostSensorHandler, failed to get nodes list: Get \"https://redacted-redacted.privatelink.canadacentral.azmk8s.io:443/api/v1/nodes\": acquiring a token for authorization header: storing the refreshed token in configuration: persisting the configuration: open C:\\Users\\Teamy\\.kube\\config.lock: The file exists."}
{"level":"warn","ts":"2023-01-03T15:09:36-05:00","msg":"failed to get policies from github release, loading policies from cache","error":"error getting: controls.json from: 'https://github.com/kubescape\\regolibrary\\releases\\latest\\download' ,error: HTTP request failed. URL: '/kubescape%5Cregolibrary%5Creleases%5Clatest%5Cdownload/controls', HTTP-ERROR: '404 Not Found', BODY: '\n\n\n\n\n\n<!DOCTYPE html>\n<html lang=\"en\" data-color-mode=\"auto\" data-light-theme=\"light\" data-dark-theme=\"dark\" data-a11y-animated-images=\"system\">\n  <head>\n    <meta charset=\"utf-8\">\n  <link rel=\"dns-prefetch\" href=\"https://github.githubassets.com\">\n  <link rel=\"dns-prefetch\" href=\"https://avatars.githubusercontent.com\">\n  <link rel=\"dns-prefetch\" href=\"https://github-cloud.s3.amazonaws.com\">\n  <link rel=\"dns-prefetch\" href=\"https://user-images.githubusercontent.com/\">\n  <link rel=\"preconnect\" href=\"https://github.githubassets.com\" crossorigin>\n  <link rel=\"preconnect\" href=\"https://avatars.githubusercontent.com\">\n\n\n\n  <link crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" href=\"https://github.githubassets.com/assets/light-719f1193e0c0.css\" /><link crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" href=\"https://github.githubassets.com/assets/dark-0c343b529849.css\" /><link data-color-theme=\"dark_dimmed\" crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" data-href=\"https://github.githubassets.com/assets/', HTTP-HEADERS: map[Cache-Control:[no-cache] Content-Security-Policy:[default-src 'none'; base-uri 'self'; block-all-mixed-content; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com objects-origin.githubusercontent.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events *.actions.githubusercontent.com wss://*.actions.githubusercontent.com online.visualstudio.com/api/v1/locations github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com objects-origin.githubusercontent.com secured-user-images.githubusercontent.com/ opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/] Content-Type:[text/html; charset=utf-8] Date:[Tue, 03 Jan 2023 20:09:36 GMT] Referrer-Policy:[origin-when-cross-origin, strict-origin-when-cross-origin] Server:[GitHub.com] Set-Cookie:[_gh_sess=; Path=/; HttpOnly; Secure; SameSite=Lax _octo=GH1.1.2102437862.1672776576; Path=/; Domain=github.com; Expires=Wed, 03 Jan 2024 20:09:36 GMT; Secure; SameSite=Lax logged_in=no; Path=/; Domain=github.com; Expires=Wed, 03 Jan 2024 20:09:36 GMT; HttpOnly; Secure; SameSite=Lax] Strict-Transport-Security:[max-age=31536000; includeSubdomains; preload] Vary:[X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With] X-Content-Type-Options:[nosniff] X-Frame-Options:[deny] X-Github-Request-Id:[C4A5:1BE2:27F610:3A8CDB:63B48B80] X-Xss-Protection:[0]], HTTP-BODY-BUFFER-LENGTH: 209299"}
{"level":"warn","ts":"2023-01-03T15:09:36-05:00","msg":"failed to get config inputs from github release, this may affect the scanning results","error":"error getting: attack_tracks.json from: 'https://github.com/kubescape\\regolibrary\\releases\\latest\\download' ,error: HTTP request failed. URL: '/kubescape%5Cregolibrary%5Creleases%5Clatest%5Cdownload/attack_tracks', HTTP-ERROR: '404 Not Found', BODY: '\n\n\n\n\n\n<!DOCTYPE html>\n<html lang=\"en\" data-color-mode=\"auto\" data-light-theme=\"light\" data-dark-theme=\"dark\" data-a11y-animated-images=\"system\">\n  <head>\n    <meta charset=\"utf-8\">\n  <link rel=\"dns-prefetch\" href=\"https://github.githubassets.com\">\n  <link rel=\"dns-prefetch\" href=\"https://avatars.githubusercontent.com\">\n  <link rel=\"dns-prefetch\" href=\"https://github-cloud.s3.amazonaws.com\">\n  <link rel=\"dns-prefetch\" href=\"https://user-images.githubusercontent.com/\">\n  <link rel=\"preconnect\" href=\"https://github.githubassets.com\" crossorigin>\n  <link rel=\"preconnect\" href=\"https://avatars.githubusercontent.com\">\n\n\n\n  <link crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" href=\"https://github.githubassets.com/assets/light-719f1193e0c0.css\" /><link crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" href=\"https://github.githubassets.com/assets/dark-0c343b529849.css\" /><link data-color-theme=\"dark_dimmed\" crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" data-href=\"https://github.githubassets.com/assets/', HTTP-HEADERS: map[Cache-Control:[no-cache] Content-Security-Policy:[default-src 'none'; base-uri 'self'; block-all-mixed-content; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com objects-origin.githubusercontent.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events *.actions.githubusercontent.com wss://*.actions.githubusercontent.com online.visualstudio.com/api/v1/locations github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com objects-origin.githubusercontent.com secured-user-images.githubusercontent.com/ opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/] Content-Type:[text/html; charset=utf-8] Date:[Tue, 03 Jan 2023 20:09:36 GMT] Referrer-Policy:[origin-when-cross-origin, strict-origin-when-cross-origin] Server:[GitHub.com] Set-Cookie:[_gh_sess=; Path=/; HttpOnly; Secure; SameSite=Lax _octo=GH1.1.291407308.1672776576; Path=/; Domain=github.com; Expires=Wed, 03 Jan 2024 20:09:36 GMT; Secure; SameSite=Lax logged_in=no; Path=/; Domain=github.com; Expires=Wed, 03 Jan 2024 20:09:36 GMT; HttpOnly; Secure; SameSite=Lax] Strict-Transport-Security:[max-age=31536000; includeSubdomains; preload] Vary:[X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With] X-Content-Type-Options:[nosniff] X-Frame-Options:[deny] X-Github-Request-Id:[C4A5:1BE2:27F653:3A8D3F:63B48B80] X-Xss-Protection:[0]], HTTP-BODY-BUFFER-LENGTH: 209479"}
{"level":"warn","ts":"2023-01-03T15:09:37-05:00","msg":"failed to get exceptions from github release, loading attack tracks from cache","error":"error getting: rules.json from: 'https://github.com/kubescape\\regolibrary\\releases\\latest\\download' ,error: HTTP request failed. URL: '/kubescape%5Cregolibrary%5Creleases%5Clatest%5Cdownload/rules', HTTP-ERROR: '404 Not Found', BODY: '\n\n\n\n\n\n<!DOCTYPE html>\n<html lang=\"en\" data-color-mode=\"auto\" data-light-theme=\"light\" data-dark-theme=\"dark\" data-a11y-animated-images=\"system\">\n  <head>\n    <meta charset=\"utf-8\">\n  <link rel=\"dns-prefetch\" href=\"https://github.githubassets.com\">\n  <link rel=\"dns-prefetch\" href=\"https://avatars.githubusercontent.com\">\n  <link rel=\"dns-prefetch\" href=\"https://github-cloud.s3.amazonaws.com\">\n  <link rel=\"dns-prefetch\" href=\"https://user-images.githubusercontent.com/\">\n  <link rel=\"preconnect\" href=\"https://github.githubassets.com\" crossorigin>\n  <link rel=\"preconnect\" href=\"https://avatars.githubusercontent.com\">\n\n\n\n  <link crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" href=\"https://github.githubassets.com/assets/light-719f1193e0c0.css\" /><link crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" href=\"https://github.githubassets.com/assets/dark-0c343b529849.css\" /><link data-color-theme=\"dark_dimmed\" crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" data-href=\"https://github.githubassets.com/assets/', HTTP-HEADERS: map[Cache-Control:[no-cache] Content-Security-Policy:[default-src 'none'; base-uri 'self'; block-all-mixed-content; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com objects-origin.githubusercontent.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events *.actions.githubusercontent.com wss://*.actions.githubusercontent.com online.visualstudio.com/api/v1/locations github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com objects-origin.githubusercontent.com secured-user-images.githubusercontent.com/ opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/] Content-Type:[text/html; charset=utf-8] Date:[Tue, 03 Jan 2023 20:09:37 GMT] Referrer-Policy:[origin-when-cross-origin, strict-origin-when-cross-origin] Server:[GitHub.com] Set-Cookie:[_gh_sess=; Path=/; HttpOnly; Secure; SameSite=Lax _octo=GH1.1.1187434972.1672776576; Path=/; Domain=github.com; Expires=Wed, 03 Jan 2024 20:09:36 GMT; Secure; SameSite=Lax logged_in=no; Path=/; Domain=github.com; Expires=Wed, 03 Jan 2024 20:09:36 GMT; HttpOnly; Secure; SameSite=Lax] Strict-Transport-Security:[max-age=31536000; includeSubdomains; preload] Vary:[X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With] X-Content-Type-Options:[nosniff] X-Frame-Options:[deny] X-Github-Request-Id:[C4A5:1BE2:27F693:3A8D99:63B48B80] X-Xss-Protection:[0]], HTTP-BODY-BUFFER-LENGTH: 209191"}
{"level":"warn","ts":"2023-01-03T15:09:37-05:00","msg":"failed to get attack tracks from github release, loading attack tracks from cache","error":"error getting: FWName_CID_CName.csv from: 'https://github.com/kubescape\\regolibrary\\releases\\latest\\download' ,error: HTTP request failed. URL: '/kubescape%5Cregolibrary%5Creleases%5Clatest%5Cdownload/FWName_CID_CName', HTTP-ERROR: '404 Not Found', BODY: '\n\n\n\n\n\n<!DOCTYPE html>\n<html lang=\"en\" data-color-mode=\"auto\" data-light-theme=\"light\" data-dark-theme=\"dark\" data-a11y-animated-images=\"system\">\n  <head>\n    <meta charset=\"utf-8\">\n  <link rel=\"dns-prefetch\" href=\"https://github.githubassets.com\">\n  <link rel=\"dns-prefetch\" href=\"https://avatars.githubusercontent.com\">\n  <link rel=\"dns-prefetch\" href=\"https://github-cloud.s3.amazonaws.com\">\n  <link rel=\"dns-prefetch\" href=\"https://user-images.githubusercontent.com/\">\n  <link rel=\"preconnect\" href=\"https://github.githubassets.com\" crossorigin>\n  <link rel=\"preconnect\" href=\"https://avatars.githubusercontent.com\">\n\n\n\n  <link crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" href=\"https://github.githubassets.com/assets/light-719f1193e0c0.css\" /><link crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" href=\"https://github.githubassets.com/assets/dark-0c343b529849.css\" /><link data-color-theme=\"dark_dimmed\" crossorigin=\"anonymous\" media=\"all\" rel=\"stylesheet\" data-href=\"https://github.githubassets.com/assets/', HTTP-HEADERS: map[Cache-Control:[no-cache] Content-Security-Policy:[default-src 'none'; base-uri 'self'; block-all-mixed-content; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com objects-origin.githubusercontent.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events *.actions.githubusercontent.com wss://*.actions.githubusercontent.com online.visualstudio.com/api/v1/locations github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com objects-origin.githubusercontent.com secured-user-images.githubusercontent.com/ opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/] Content-Type:[text/html; charset=utf-8] Date:[Tue, 03 Jan 2023 20:09:37 GMT] Referrer-Policy:[origin-when-cross-origin, strict-origin-when-cross-origin] Server:[GitHub.com] Set-Cookie:[_gh_sess=; Path=/; HttpOnly; Secure; SameSite=Lax _octo=GH1.1.1120685424.1672776577; Path=/; Domain=github.com; Expires=Wed, 03 Jan 2024 20:09:37 GMT; Secure; SameSite=Lax logged_in=no; Path=/; Domain=github.com; Expires=Wed, 03 Jan 2024 20:09:37 GMT; HttpOnly; Secure; SameSite=Lax] Strict-Transport-Security:[max-age=31536000; includeSubdomains; preload] Vary:[X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With] X-Content-Type-Options:[nosniff] X-Frame-Options:[deny] X-Github-Request-Id:[C4A5:1BE2:27F6CC:3A8DD7:63B48B80] X-Xss-Protection:[0]], HTTP-BODY-BUFFER-LENGTH: 209587"}
{"level":"info","ts":"2023-01-03T15:09:37-05:00","msg":"Downloading/Loading policy definitions"}
{"level":"fatal","ts":"2023-01-03T15:09:37-05:00","msg":"unknown policy kind"}
```

This part stands out to me
```
{"level":"warn","ts":"2023-01-03T15:09:36-05:00","msg":"failed to get policies from github release, loading policies from cache","error":"error getting: controls.json from: 'https://github.com/kubescape\\regolibrary\\releases\\latest\\download' ,error: HTTP request failed. URL: '/kubescape%5Cregolibrary%5Creleases%5Clatest%5Cdownload/controls', HTTP-ERROR: '404 Not Found'
```

From CLI output without verbose it shows the symbols as using backslashes
```
[warning] failed to get policies from github release, loading policies from cache. error: error getting: exceptions.json from: 'https://github.com/kubescape\regolibrary\releases\latest\download' ,error: HTTP request failed. URL: '/kubescape%!C(MISSING)regolibrary%!C(MISSING)releases%!C(MISSING)latest%!C(MISSING)download/exceptions', HTTP-ERROR: '404 Not Found',
```

Digging I found this file

https://github.com/kubescape/opa-utils/blob/master/gitregostore/gitstoreutils.go#L66

and I noticed

![image](https://user-images.githubusercontent.com/9356891/210435782-c8b22556-b214-4713-af57-db70410f3ddf.png)

and sure enough in the PR notes

https://github.com/kubescape/opa-utils/pull/69

> * using url join instead of path join

and there's [a release for the opa-utils with the hotfix](https://github.com/kubescape/opa-utils/releases/tag/v0.0.223) already, but not a kubescape release yet.

I typed all this already so I'll submit it anyways while waiting for the kubescape repo release with the fix lol

**Comment by Bezbran** [comment]
Hi @TeamDman,
Looks like the hotfix released just on time [v2.0.181:](https://github.com/kubescape/kubescape/releases/tag/v2.0.181)
https://github.com/kubescape/kubescape/pull/1002
You can see here the build workflow:
https://github.com/kubescape/kubescape/actions/runs/3832828403/jobs/6523746020
Can you please verify it?

**Comment by TeamDman** [comment]
Working on v2.0.181, thanks!!
